### DISCUSSION TITLE
Can rauc be used to switch between yocto and buildroot?

### DISCUSSION THREAD

**Discussion by synaption** [rauc/rauc#1036] [Q&A] [answered]
Can RAUC be used to switch from buildroot to yocto or vice versa?  I don't see any reason why not, but I don't understand everything about the update bundle creation process.  Just curious.

**Comment by ejoerns** [comment, marked as answer]
Absolutely, yes. For RAUC, which just installs file system images, it is not relevant which Linux distro is inside the image and both Yocto and buildroot already have support for integrating RAUC in the target system as well as for generating RAUC bundles.

The creation of bundles is also not strictly coupled to a build system.  Even if this is actually the recommended way, using a standalone RAUC tool (e.g. from your desktop distro) would work as well.

All you have to do is to make sure that the system.conf integrated in both systems has the same slot layout and that you have the necessary (signing) keys in both infrastructures.

**Reply by jluebbe** [reply]
@synaption Does this answer your question?

**Reply by synaption** [reply]
pretty much what I thought ya.
